### PR TITLE
Adds the Front End of the Create Post option for StackOverflow feature

### PR DESCRIPTION
**Pull request by PanwarJayant**
Please review the changes.

### CODE REVIEW

**Review by Dayitva — APPROVED** [review]
LGTM

**Comment by prajneya** [comment]
LGTM. Will merge this after the Single Post merge

**Review by prajneya — APPROVED** [review]

**Pull request merged**
